internal/install/static_snapshot_yml.go

```diff
@@ -75,25 +75,41 @@ services:
       package-registry:
         condition: service_healthy
 
-  elastic-agent:
+  fleet-server:
     image: ${ELASTIC_AGENT_IMAGE_REF}
     depends_on:
       elasticsearch:
         condition: service_healthy
       kibana:
         condition: service_healthy
+    healthcheck:
+      test: "curl -f http://127.0.0.1:8220/api/status | grep HEALTHY 2>&1 >/dev/null"
```

Comment:
Can add `--insecure` to the `curl` command and change to `https` if you remove the `FLEET_SERVER_INSECURE_HTTP` below.